confed/util.go

```diff
@@ -32,25 +34,29 @@ func runCommand(captureStdout bool, stdin io.Reader, command string, args ...str
 			if ok {
 				status = ws.ExitStatus()
 			}
-			return nil, fmt.Errorf("exit status %d from %s %s: %s",
+			err = fmt.Errorf("exit status %d from %s %s: %s",
 				status, command, strings.Join(args, " "),
-				string(stderr.Bytes()))
+				string(res.stderr.Bytes()))
 		}
-		return nil, err
 	}
 
-	return stdout, nil
+	return
 }
 
-func extPreprocess(commandAndArgs []string, in []byte) (*bytes.Buffer, error) {
+func extPreprocess(commandAndArgs []string, in []byte) (RunCommandResult, error) {
 	if len(commandAndArgs) < 1 {
-		return nil, errors.New("commandAndArgs must not be empty")
+		return RunCommandResult{}, errors.New("commandAndArgs must not be empty")
 	}
 
 	return runCommand(true, bytes.NewBuffer(in), commandAndArgs[0], commandAndArgs[1:]...)
 }
 
-func loadConfigBytes(path string, preprocessCmd []string) (bs []byte, err error) {
+type LoadConfigResult struct {
+	content []byte
+	preprocessErrors string
```

Comment:
```suggestion
	preprocessorErrors string
```

а то тут глагол затесался как будто

Reply:
Заменил